examples/meteor/client/templates/todo.js

```diff
@@ -0,0 +1,60 @@
+Template.todo.onCreated(function () {
+	// 'this' refers to the template instance within onCreated() and onRendered()
+	this.editing = new ReactiveVar(false);
+	this.previousText = new ReactiveVar(null);
+});
+
+// When the edit field is shown, set focus to the input and move the cursor
+// to the end
+Template.todoEdit.onRendered(function () {
+	var inputField = this.find('input');
+	var inputLength = inputField.value.length;
+
+	inputField.focus();
+	inputField.setSelectionRange(inputLength, inputLength);
+})
+
+Template.todo.events({
+	'click input.toggle': function () {
+		Todos.update(this._id, {$set: {completed: !this.completed}});
+	},
+
+	'dblclick label': function (event, template) {
+		template.editing.set(true);
+		template.previousText.set($(event.target).text());
+	},
+
+	'click button.destroy': function () {
+		Todos.remove(this._id);
+	},
+
+	'keyup li.editing input.edit, focusout li.editing input.edit': function (event,
+		template) {
+		if ((event.type === 'keyup' && event.which === 13) ||
```

Comment:
keycodes should use a constant (https://github.com/tastejs/todomvc/blob/master/app-spec.md#code)